languages/en-CA.js

```diff
@@ -1,15 +1,173 @@
+			RESOLVER_INVALID_ROLE: (name) => `${name} must be a role mention or role id.`,
+			RESOLVER_INVALID_STRING: (name) => `${name} must be a valid string.`,
+			RESOLVER_INVALID_TIME: (name) => `${name} must be a valid duration or date string.`,
+			RESOLVER_INVALID_URL: (name) => `${name} must be a valid url.`,
+			RESOLVER_INVALID_USER: (name) => `${name} must be a mention or valid user id.`,
+			RESOLVER_STRING_SUFFIX: ' characters',
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} must be exactly ${min}${suffix}.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} must be between ${min} and ${max}${suffix}.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} must be greater than ${min}${suffix}.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} must be less than ${max}${suffix}.`,
+			REACTIONHANDLER_PROMPT: 'Which page would you like to jump to?',
+			COMMANDMESSAGE_MISSING: 'Missing one or more required arguments after end of input.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} is a required argument.`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Missing a required option: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Your option didn't match any of the possibilities: (${possibles})`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time, abortOptions) => `${tag} | **${error}** | You have **${time}** seconds to respond to this prompt with a valid argument. Type **${abortOptions.join('**, **')}** to abort this prompt.`,
+			// eslint-disable-next-line max-len
+			MONITOR_COMMAND_HANDLER_REPEATING_REPROMPT: (tag, name, time, cancelOptions) => `${tag} | **${name}** is a repeating argument | You have **${time}** seconds to respond to this prompt with additional valid arguments. Type **${cancelOptions.join('**, **')}** to cancel this prompt.`,
+			MONITOR_COMMAND_HANDLER_ABORTED: 'Aborted',
+			INHIBITOR_MISSING_BOT_PERMS: (missing) => `Insufficient permissions, missing: **${missing}**`,
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `The guild is missing the **${settings.join(', ')}** guild setting${settings.length !== 1 ? 's' : ''} and thus the command cannot run.`,
+			INHIBITOR_RUNIN: (types) => `This command is only available in ${types} channels.`,
+			INHIBITOR_RUNIN_NONE: (name) => `The ${name} command is not configured to run in any channel.`,
+			COMMAND_BLACKLIST_DESCRIPTION: 'Blacklists or un-blacklists users and guilds from the bot.',
+			COMMAND_BLACKLIST_SUCCESS: (usersAdded, usersRemoved, guildsAdded, guildsRemoved) => [
+				usersAdded.length ? `**Users Added**\n${util.codeBlock('', usersAdded.join(', '))}` : '',
+				usersRemoved.length ? `**Users Removed**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**Guilds Added**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['util' is not defined.](https://app.codacy.com/manual/Butterstroke/MargarineBot/pullRequest?prid=5457955)